neo/io/axonio.py

```diff
@@ -296,7 +296,7 @@ def read_block(self, lazy=False, cascade=True):
 
                     anaSig = AnalogSignal(signal, sampling_rate=sampling_rate,
                                           t_start=t_start,
-                                          name=name.decode('utf-8'),
+                                          name=str(name),

```

Comment:
Yes. it would be better. It is not a big API break.
The thing is that we need to hack the test engine which test that each attribute have the good type (only one).
If we change AnalogSignal.name to unicode we need to change for all IOs. The better would to allow several types for each attributes whihch is not the case see AnalogSignal._necessary_attrs and BaseNeo._necessary_attrs.